robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -0,0 +1,53 @@
+import time
+
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from locators_46 import npsp_lex_locators
+from logging import exception
+
+@pageobject("Custom", "NPSPSettings")
+class NPSPSettingsPage(BasePage):
+    object_name = None
+
+    
+    @property
+    def npsp(self):
+        return self.builtin.get_library_instance('NPSP')
+    
+    
+    def _go_to_page(self, filter_name=None):
+        """To go to NPSP Settings page"""
+        url= self.cumulusci.org.lightning_base_url+"/lightning/n/NPSP_Settings"
+        self.selenium.go_to(url)
+        self.salesforce.wait_until_loading_is_complete()
+        self.npsp.wait_for_locator("frame","Nonprofit Success Pack Settings")
+        self.npsp.choose_frame("Nonprofit Success Pack Settings")
+        
+    def open_main_menu(self,title): 
+        """Waits for the menu item to load and clicks to expand menu""" 
+        self.selenium.wait_until_page_contains("System Tools")  
```

Comment:
You can add a custom error message here using the "error=" parameter. In future, if System Tools is not available, the error message would be very clear to debug